features/admin_panel/adding_user_to_admin_panel.feature

```diff
@@ -0,0 +1,35 @@
+Feature: Adding a user to admin panel
```

Comment:
It could be good to add unhappy paths scenarios (what happen if email will be empty or invalid).